developer_manual/core/theming.rst

```diff
@@ -331,4 +340,5 @@ If you think a new section should be added to core however, please create a PR w
 .. _Safari: https://developer.apple.com/safari/tools/
 .. _the guide on Can I Use: http://caniuse.com/#feat=css-gradients
 .. _this example: https://github.com/owncloud/core/blob/master/lib/private/Settings/SettingsManager.php#L195   
-
+.. _signed: https://doc.owncloud.org/server/10.0/developer_manual/app/code_signing.html
```

Comment:
Use a ref: link instead of absolute link which needs to be updated after every new release or move/rename of the file?